src/core_modules/capture-core/components/WidgetEnrollment/TransferModal/OrgUnitField/useSearchScopeWithFallback.js

```diff
@@ -17,6 +17,8 @@ export const useSearchScopeWithFallback = ({ searchText }: Props) => {
         },
         {
             enabled: !searchText,
+            // Clearing cache after 120 minutes to avoid memory leaks
+            cacheTime: 120 * 60 * 1000,
```

Comment:
This query has a constant query key, so no problem with keeping the default `cacheTime` here.

Reply:
👍